plugin/pom.xml

```diff
@@ -46,6 +46,22 @@
   </developers>
 
   <dependencies>
+    <dependency>
+      <groupId>org.everit.json</groupId>
+      <artifactId>org.everit.json.schema</artifactId>
+      <version>1.3.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.5</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.dataformat</groupId>
+      <artifactId>jackson-dataformat-yaml</artifactId>
+      <version>2.7.7</version>
```

Comment:
Is this required?

Reply:
No I will get rid of this probably in the cleaning of commits.

Reply:
` public static String convertToJson(String yamlString) {
        Yaml yaml= new Yaml();`

Reply:
Can we not use the already shaded snake yaml?

Reply:
I am not quite sure if that generates a yaml object but, as far as I tried it errored out.